### PR TITLE
VAR-173 | Maximum duration of reservations should not apply to staff

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: src/domain/resource/utils.js [hunk 1]
```diff
@@ -401,7 +401,7 @@ export const isTimeRangeReservable = (resource, start, end) => {
 
   // Reservation cannot be longer than the resources max period if max period is set.
   const maxPeriod = get(resource, 'max_period', null);
-  if (maxPeriod) {
+  if (!isStaff && maxPeriod) {
```
Comment (midori0507) [inline src/domain/resource/utils.js]: Please add comment for this exception: 
Something as `As a staff, user have permission to ...`

File: src/domain/resource/utils.js [hunk 2]
```diff
@@ -434,7 +434,11 @@ export const isTimeRangeReservable = (resource, start, end) => {
  * @param resource {object} Resource object.
  * @returns {boolean}
  */
-export const isFullCalendarEventDurationEditable = (resource) => {
+export const isFullCalendarEventDurationEditable = (resource, start, end, isStaff = false) => {
+  if (isStaff) {
+    return true;
```
Comment (midori0507) [inline src/domain/resource/utils.js]: Same comment here

File: src/domain/resource/utils.js [hunk 3]
```diff
@@ -383,7 +383,7 @@ export const isDateReservable = (resource, date) => {
  * @param end {Date|string} Either a Date object or date string that can be parsed as moment object.
  * @returns {boolean}
  */
-export const isTimeRangeReservable = (resource, start, end) => {
+export const isTimeRangeReservable = (resource, start, end, isStaff = false) => {
```
Comment (midori0507) [inline src/domain/resource/utils.js]: Update / new unit test will be nice.

File: src/domain/resource/utils.js (hunk 1 above)
Comment (badrange) [reply src/domain/resource/utils.js]: I added the code comment to the function dockblock , that should be more correct right?

File: src/domain/resource/utils.js (hunk 2 above)
Comment (badrange) [reply src/domain/resource/utils.js]: I added the code comment to the function dockblock , that should be more correct right?

File: src/domain/resource/utils.js (hunk 3 above)
Comment (badrange) [reply src/domain/resource/utils.js]: Done!

**Pull request merged**